doc/src/find_and_immediate.md

```diff
@@ -158,3 +158,18 @@ If the user's architecture is not listed above they will need to run the Find AP
 ### Backend Limitations
 
 OpenCL support for immediate mode via the fallback is limited to fp32 datatypes. This is because this current release's fallback path goes through GEMM which on the OpenCL is serviced through MIOpenGEMM -- which itself only contains support for fp32. The HIP backend uses rocBLAS as its fallback path which contains a richer set of datatypes.
+
+
+### Find Modes
+
+MIOpen provides a set of Find modes which are used to accelerate the Find calls. The different modes are set by using the environment variable `MIOPEN_FIND_MODE`, and setting it to one of the values:
+
+- `NORMAL`: Normal Find: This is the full Find mode call, which will benchmark all the solvers and return a list.
+- `FAST`: Fast Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, utilize the Immediate mode fallback. If Start-up times are expected to be faster, but worse GPU performance.
+- `HYBRID`: Hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery. Slower start-up times but none GPU performance drop at all.
```

Comment:
```suggestion
- `HYBRID` or `3` or variable is unset: Hybrid Find: Checks the [Find-Db](https://rocmsoftwareplatform.github.io/MIOpen/doc/html/finddb.html) for an entry. If there is a Find-Db hit, use that entry. If there is a miss, use the existing Find machinery. Slower start-up times but none GPU performance drop at all.
```

Reply:
also `none` to `no`

Reply:
Yes. And `Slower start-up times but...` -> `Slower start-up times than Fast Find but...`